usagov/usagov-2021
# USAGOV-1094-Update Referrer-Policy in HTTP header to help reporting in Search.gov

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1094

## Description

Updated our referrer policy based [on these guidelines](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy).

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- On a few pages check to make sure that the meta tag no-referrer-when-downgrade is set (<meta name="referrer" content="no-referrer-when-downgrade">)

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
